### PR TITLE
Airflow 2.8.1 has been released

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

File: newsfragments/config.toml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # under the License.
 [tool.towncrier]
 name = "Airflow"
-filename = "../RELEASE_NOTES.rst"
+filename = "RELEASE_NOTES.rst"
```
Comment (dirrao) [inline newsfragments/config.toml:19]: I don't see the change in the RELEASE_NOTES.rst doc path. Any reason why its path changed here?

**Review by potiuk — APPROVED** [review]

File: newsfragments/config.toml (hunk 1 above)
Comment (potiuk) [reply newsfragments/config.toml:19]: I guess it's the matter where you run towncrier (my guess).

Comment (jedcunningham) [reply newsfragments/config.toml:19]: Yeah, that's exactly it.

Comment (jedcunningham) [reply newsfragments/config.toml:19]: This allows the towncrier step [here](https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md) to work from the repo root.

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**